src/appimagetool.c

```diff
@@ -194,8 +194,19 @@ int sfs_mksquashfs(char *source, char *destination, int offset) {
             args[i++] = exclude_file;
         }
 
+        args[i++] = "-mkfs-fixed-time";
```

Comment:
Would you do this _all the time_ or only in certain circumstances?

Reply:
See the discussion in the issue.